src/index/Index.h

```diff
@@ -307,6 +307,39 @@ class Index {
 
   bool hasAllPermutations() const { return _spoFile.isOpen(); }
 
+  const IndexMetaDataHmap& getPsoMeta() const {
+    if (hasAllPermutations()) {
+      return _psoMeta;
+    } else {
+      AD_THROW(ad_semsearch::Exception::CHECK_FAILED,
+               "Can only use the spo metadta if all 6 permutations "
```

Comment:
Is this true? I mean we still have `_psoMetah`?

Reply:
I just forgot to remove the check after copying the method. It's gone now.